tool/tsh/tsh.go

```diff
@@ -760,13 +792,64 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 		bench.Hidden()
 	}
 
+	var err error
+
+	cf.executablePath, err = os.Executable()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	// configs
+	setEnvFlags(&cf, os.Getenv)
+
+	confOptions, err := loadAllConfigs(cf)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	cf.TshConfig = *confOptions
+
+	// aliases
+	ar := newAliasRunner(cf.TshConfig.Aliases)
+	aliasCommand, runtimeArgs := findAliasCommand(args)
+	if aliasDefinition, ok := ar.getAliasDefinition(aliasCommand); ok {
+		err = ar.markAliasSeen(aliasCommand)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+		newArgs, err := expandAliasDefinition(aliasDefinition, runtimeArgs)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+		err = ar.runAliasCommand(ctx, cf.executablePath, newArgs[0], newArgs[1:])
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+		return nil
```

Comment:
Nit: I would combine this into alias runner's `Run()` method or something.

Reply:
Done.